erts/emulator/test/trace_call_time_SUITE.erl

```diff
@@ -96,7 +96,7 @@ all() ->
 %% Tests basic call time trace
 basic(Config) when is_list(Config) ->
     P = erlang:trace_pattern({'_','_','_'}, false, [call_time]),
-    M = 1000,
+    M = 900,
```

Comment:
Does this mean I caused a regression while generalising call_time and call_memory, or it was flaky before?

Reply:
You provoked a bug that was already there.
The test sets `max_heap_size` (don't know why?) which was never exceeded. But the increase of the extra stack frame used by call_time (and now also call_memory) caused the `max_heap_size` to be exceeded if run with the `+JPperf true` command line option. And that in turn exposed some bugs when `max_heap_size` is killing the process, which I have now fixed in #6816.